api/v1/config_types.go

```diff
@@ -418,12 +425,22 @@ type ConfigComponentUpgradeReport struct {
 	Interval ReporterInterval `json:"interval,omitempty"`
 	// +optional
 	Criteria ReporterCriteria `json:"criteria,omitempty"`
+	// +optional
+	ExtraMessage ReporterExtraMessage `json:"extraMessage,omitempty"`
+}
+
+// ConfigActivePromotionReport defines a configuration of active promotion report
+type ConfigActivePromotionReport struct {
+	// +optional
+	ExtraMessage ReporterExtraMessage `json:"extraMessage,omitempty"`
 }
 
 // ConfigPullRequestTrigger defines a configuration of pull request trigger report
 type ConfigPullRequestTriggerReport struct {
 	// +optional
 	Criteria ReporterCriteria `json:"criteria,omitempty"`
+	// +optional
+	ExtraMessage ReporterExtraMessage `json:"extraMessage,omitempty"`
```

Comment:
Could we use `string` instead of `ReporterExtraMessage` since we haven't checked the type or declared any var from this type?